Tools/ros2/ardupilot_dds_tests/test/ardupilot_dds_tests/test_mode_switch.py

```diff
@@ -0,0 +1,72 @@
+import rclpy
+import time
+from ardupilot_msgs.srv import ModeSwitch
+from enum import Enum
+
+
+class FlightModes(Enum):
```

Comment:
Are these copter specific?

Reply:
I have added all the flight modes given here - https://github.com/ArduPilot/MAVProxy/blob/c4619af111761135408476d6a7e79cea68cf8388/MAVProxy/modules/mavproxy_chat/assistant_setup/copter_flightmodes.txt

Should I only include the copter modes?

Reply:
Well that list is copter-specific in the title, so perhaps the class  could be called `FlightModesCopter`.  It does not have FBWA, Thermal, Cruise, or any of the other plane-specific modes.